presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/QueryBuilder.java

```diff
@@ -253,7 +248,7 @@ private String toPredicate(String columnName, String operator, Object value, Jdb
 
     private String quote(String name)
     {
-        return identifierQuote + name.replace(identifierQuote, identifierQuote + identifierQuote) + identifierQuote;
+        return customization.quote(name);
```

Comment:
inline the method

Reply:
I disagree. It makes the diff and code longer for no benefit. The fact that we already had a method allows us to cleanly encapsulate this change.